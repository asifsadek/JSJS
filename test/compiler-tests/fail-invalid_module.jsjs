val l = ListLabels.length([1,2,3,4]);

print(l);
