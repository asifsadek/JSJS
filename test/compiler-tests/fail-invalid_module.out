Type error: Module 'ListLabels' not defined
